cmake/developer_package/compile_flags/os_flags.cmake

```diff
@@ -268,7 +268,7 @@ macro(ov_arm_sve_optimization_flags flags)
         if(ANDROID_ABI STREQUAL "arm64-v8a")
             set(${flags} -Wno-unused-command-line-argument)
             if(CXX_SVE_FOUND)
-                list(APPEND ${flags} -march=armv8-a+sve)
+                list(APPEND ${flags} -march=armv8.2-a+sve)
```

Comment:
Could you please clarify why march is changed?

Reply:
https://github.com/openvinotoolkit/openvino/blob/d8d5ef9a2dbeba735bed7e2c98bc520e4b349238/cmake/developer_package/compile_flags/os_flags.cmake#L236-L242
For ARM neon  -march=armv8.2a is chosen.
I have made the same change for SVE arch as well , otherwise I was getting build error stating "target architecture mismatch"  for Neon based code.

Reply:
Then I guess we need to call ov_check_compiler_supports_sve with -march=armv8.2-a+sve as well?